### ISSUE TITLE
Change text color in tab

### ISSUE THREAD

**Issue by indi110** [tablacus/TablacusExplorer#567]
Hi, I wanted to change the text color for tabs if the filepath included the word "special".
This is using "User style sheet".
This is an attempt.

```
*special* .namecell {
  color: blue
}
```

YES - colored blue
C:\Test\special 1
C:\Test\special case

NO
C:\Test\new folder

**Comment by ccfs** [comment]
"User style sheet" is an application-level styling tool using **CSS**. It cannot be associated with any user data.
The easiest approach to your request is to enhance "Tab color plus" by adding text color to each record. If specified, text color will be used regardless of tab color. "Tab list" may also need be enhanced to show the text color.

Regards,

**Comment by indi110** [comment]
OK thanks, how can I enhance this?
I'm aware now that "Tab color plus" takes information from this file: "tabcolorplus.tsv" in the "config" folder.

If I open using Notepad, I can see as per below.
How can I add to this to change text color?

```
*folder 1*	#FFFF99
*folder 2*	#FFE1FF
*folder 3*	#99FFCC
```

**Comment by indi110** [comment]
Ok thanks.
I actually prefer to edit the config file directly though. I set up a spreadsheet and copy and paste the data, rather than using the menu. It makes it easier for me.

**Comment by indi110** [comment]
Hi, just wondering if this is available.

**Comment by indi110** [comment]
I know I've been asking a lot of questions but hopefully this is the last one.
Will this be possible?
Thank you.

**Comment by tablacus** [comment]
I think it is possible with some add-ons modifications.
What purpose do you have with it?
I think the design of the program will change accordingly.

Regards,

**Comment by indi110** [comment]
Hi, thanks for responding.

The purpose is as follows:
**I have a several tabs open for different projects. I'm using background colour to differentiate between projects. If there was another option to have the text colour changed, then I could see which part of the project the tab belonged to.**

Initially it got really confusing with many tabs open. "Tab color plus" helped with this a lot.
If there was another differentiator then this would help even more. Changing the text colour would do this.
Or maybe by either underlining or making bold that may help as well.

So basically aiming for 2 differentiators. One is tab colour background using "Tab color plus". Second is either text colour or by underlining text or making text bold ... or a combination of these.

**Comment by tablacus** [comment]
Thank you for sharing your purpose to that I got more motivation. 
Do you use the tab group add-on?
It's useful to change the group for each project.

Regards,

**Comment by indi110** [comment]
Hi, I'm not on my work computer now ... but I think I do use it.
So far I have 3 groups, one of which is Projects.
I guess I could use more groups as another way.

**Comment by duarteframos** [comment]
>  I have a several tabs open for different projects. I'm using background colour to differentiate between projects.

You may want to consider also using [Tab Groups](https://github.com/tablacus/TablacusExplorerAddons/wiki/tabgroups) addon to categorize different projects into separate tabs, possibly even save them into layouts savable to files you can easily restore.

**Comment by indi110** [comment]
OK thanks I will try this as well.

**Comment by tablacus** [comment]
I have adjusted "Tab plus" and "Tab color plus".
Please update and give it a try.

Regards,

**Comment by indi110** [comment]
Many thanks tablacus, this is working.
Cheers.

**Comment by indi110** [comment]
Hope your software is more accessible.
